src/components/Select/Select.tsx

```diff
@@ -47,13 +50,15 @@ export const Select = <T extends string, TFieldValues extends FieldValues>({
         }
         onChange(event)
       }}
+      value={value?.label}
```

Comment:
I don't get this, could you say why `value` is set to `value.label`?

Reply:
This is for applying a value indirect, I'll refactor it.